src/components/Card/Card.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {block} from '../utils/cn';
+
+import './Card.scss';
+
+const b = block('card');
+
+type SelectionCardView = 'outlined' | 'clear';
+type ContainerCardView = 'outlined' | 'filled' | 'raised';
+
+type ContainerCardTheme = 'normal' | 'info' | 'positive' | 'warning' | 'danger';
+
+export type CardType = 'selection' | 'action' | 'container';
+export type CardTheme = ContainerCardTheme;
+export type CardView = SelectionCardView | ContainerCardView;
+
+export interface CardProps {
+    children: React.ReactNode;
+    className?: string;
+    /** Card click handler. Available for type: 'selection', 'action' */
+    onClick?: (event: React.MouseEvent<HTMLDivElement>) => void;
+    /** Disabled card. Available for type: 'selection', 'action' */
+    disabled?: boolean;
+    /** Selected card. Available for type: 'selection' */
+    selected?: boolean;
+    /** Card's type affects on available properties */
+    type?: CardType;
+    /** Available for type: 'container' and 'selection' */
+    view?: CardView;
+    /** Card's base color. Available for type: 'container' */
+    theme?: ContainerCardTheme;
+}
+
+export const Card: React.FC<CardProps> = ({
```

Comment:
Using React.FC introduces some inconvenience in use and not recommended, could you refactor to normal function?

Reply:
8e70d81